Gemfile

```diff
@@ -41,15 +49,16 @@ end
 
 group :test do
   gem "poltergeist"
+  gem "capybara", ">= 2.6.2"
   gem "database_cleaner"
-  gem "shoulda-matchers", require: false
+  gem "shoulda-matchers"
   gem "simplecov", require: false
-  gem "sinatra"
+  gem "sinatra", github: "sinatra/sinatra"
```

Comment:
I'd leave comments above each of these saying why you're not on the published gem (or a link to an issue or whatever), just so people know what to look for when upgrading.